src/parser/rules.rs

```diff
@@ -328,7 +328,24 @@ impl Parser {
             }
             // 5
             TokenKind::Literal(Value::Int) => {
-                Expression::Int(token.raw.parse::<u32>().map_err(|e| e.to_string())?)
+                // Ignore spacing character (E.g. 1_000_000)
+                let clean_str = token.raw.replace('_', "");
```

Comment:
Isn't lexer ignoring it already?

Reply:
Nope. See `eat_digits` method.

Reply:
https://github.com/antimony-lang/antimony/blob/e8b9c64cc4f623962bf47a4b99665a03c3967c8b/src/lexer/mod.rs#L406-L408

Reply:
It doesn't skip, it eats them, just like numbers:

https://github.com/antimony-lang/antimony/blob/e8b9c64cc4f623962bf47a4b99665a03c3967c8b/src/lexer/mod.rs#L409-L412

Admittedly, this could be cleaner. But I didn't know better at the time.😅